### PR TITLE
docs: `integrations` reference updates 3

### PR DESCRIPTION
**Pull request by leo-gan**
Added missed provider pages and links. Fixed inconsistent formatting.

### CODE REVIEW

**Comment by leo-gan** [comment]
@efriis Erick, could you, please, make the lint error more informative? Thanks!
With the current error, I cannot localize what is wrong.
The error is:

> Traceback (most recent call last):
>   File "/home/runner/work/langchain/langchain/docs/scripts/check_templates.py", line 95, in <module>
> Checking llms page /home/runner/work/langchain/langchain/docs/docs/integrations/llms/rwkv.mdx
>     main(*sys.argv[1:])
>   File "/home/runner/work/langchain/langchain/docs/scripts/check_templates.py", line 89, in main
>     check_header_order(path)
>   File "/home/runner/work/langchain/langchain/docs/scripts/check_templates.py", line 80, in check_header_order
>     raise ValueError(
> ValueError: Document /home/runner/work/langchain/langchain/docs/docs/integrations/llms/rwkv.mdx does not match the expected header order. Please see https://github.com/langchain-ai/langchain/issues/24803 for instructions on how to correctly format a llms integration page.

**Comment by leo-gan** [comment]
@hwchase17 It is tempting to enforce a template to new examples. But it doesn't work if the template is not tested on the old examples. **I found no old LLM example that fits this new template.** How about "eating dog food" before enforcing?

**Comment by efriis** [comment]
Hey @leo-gan ! We're only enforcing this for newly contributed docs. The linked issue in the error message outlines exactly how to bootstrap this document and the [template to match](https://github.com/langchain-ai/langchain/blob/master/libs/cli/langchain_cli/integration_template/docs/llms.ipynb)

Let me know if that doesn't make sense

**Comment by leo-gan** [comment]
> Hey @leo-gan ! We're only enforcing this for newly contributed docs. The linked issue in the error message outlines exactly how to bootstrap this document and the [template to match](https://github.com/langchain-ai/langchain/blob/master/libs/cli/langchain_cli/integration_template/docs/llms.ipynb)
> 
> Let me know if that doesn't make sense

@efriis Erick, 
1. a template is the template, if it is repeated in most documents. Only then does it start to help the user (because the user is working now with a consistent format). If we have 90% of examples in free format, the template is useless. Let's apply the template on a significant portion of documents and only then enforce this template.
2. I'm sure that after applying this template to 5-10 documents, this template will change, and the template enforcement code will change. Enforcing this template now is premature.
3. The existing LLM template proposes a heading that, if applied to all documents, requires significant changes in existing documents (not only the head renaming). Some examples are simple, some are complex and detailed. Simple examples are better with the subset of the template headers, complex examples are better with supersets. The strict heading enforcement is too limiting.

**Comment by efriis** [comment]
Appreciate the feedback. Which heading is creating the most issues for you?

Updated the error message to be more clear which headings are missing in #25782

**Comment by leo-gan** [comment]
@efriis "Setup" and "Installation" from the template is usually a single section "Setup and installation" or "Installation and Setup". Both should be OK, because it is clear for the reader. 
The "Overview" section is usually a text after page Title and the "Installation/Setup" section. No special header
The "Instantiation" section is usually one-liner and a part of any other sections with code, but I like when it is in a separate section. It emphasizes the usage of this specific class/function.
The "Invocation" is usually the "Example" or several sections with examples. It is very flexible. See [the example](https://python.langchain.com/v0.2/docs/integrations/document_loaders/url). If we enforce heading, we should completely refactor this example, not just rename titles and move pieces of the document.
Here is [another example](https://python.langchain.com/v0.2/docs/integrations/vectorstores/elasticsearch/).
Here are the LLM examples: [1,](https://python.langchain.com/v0.2/docs/integrations/llms/azure_openai/) , [2](https://python.langchain.com/v0.2/docs/integrations/llms/openai/)

To summarize: 
- General description [under the page title] can be enforced
- generalizing Installation and Setup [under one section] - can be enforced.
- enforcing anything else requires a total refactoring of all examples, which is not feasible and brings too little to usability/readability/comprehension

**Comment by efriis** [comment]
got it, and thank you for writing up. Let's stick with the templates for now (and fix this PR if it's desired), as we'd like to standardize the format a bit more than you outlined.

**Comment by leo-gan** [comment]
Righ now the error:
ValueError:

>  Document /home/runner/work/langchain/langchain/docs/docs/integrations/llms/rwkv.mdx is missing headers:
> - ### Integration details
> - ### Credentials
> - ### Installation
> - ## Chaining
> - ## API reference

- do we need to enforce H3+ sections? (### )
- We use LLMs in many components not only in chains, so why this "## Chaining" section?
- "## API reference" We have the APR Reference automatically in the "Instantiation" section. For example, 
`API Reference:` [OpenAI](https://python.langchain.com/v0.2/api_reference/openai/llms/langchain_openai.llms.base.OpenAI.html) 
Manually added link is not maintainable and confuses users (Why do we have two API Reference links? Do they equal? What is the difference?)

File: docs/docs/integrations/providers/ainetwork.mdx [hunk 1]
```diff
@@ -13,6 +13,22 @@ You need to install `ain-py` python package.
 pip install ain-py
 ```
 You need to set the `AIN_BLOCKCHAIN_ACCOUNT_PRIVATE_KEY` environmental variable to your AIN Blockchain Account Private Key.
+
+## Tools
+
+Tools that help you interact with the `AINetwork` blockchain. They are all included
+in the `AINetworkToolkit` toolkit.
+
+See a [usage example](/docs/integrations/toolkits/ainetwork).
```
Comment (efriis) [inline docs/docs/integrations/providers/ainetwork.mdx:22]:
```suggestion
See a [usage example](/docs/integrations/tools/ainetwork).
```

File: docs/docs/integrations/providers/amadeus.mdx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+```bash
+pip install amadeus
+```
+
+## Tools
+
+Tools that help you interact with the `Amadeus travel APIs`. They are all included
+in the `Amadeus` toolkit.
+
+See a [usage example](/docs/integrations/toolkits/amadeus).
```
Comment (efriis) [inline docs/docs/integrations/providers/amadeus.mdx:21]:
```suggestion
See a [usage example](/docs/integrations/tools/amadeus).
```

File: docs/docs/integrations/providers/slack.mdx [hunk 3]
```diff
@@ -15,6 +15,20 @@ See a [usage example](/docs/integrations/document_loaders/slack).
 from langchain_community.document_loaders import SlackDirectoryLoader
 ```
 
+## Tools
+
+Tools that help you interact with the `Slack`. They are all included
+in the `SlackToolkit` toolkit.
+
+See a [usage example](/docs/integrations/toolkits/slack).
```
Comment (efriis) [inline docs/docs/integrations/providers/slack.mdx:23]:
```suggestion
See a [usage example](/docs/integrations/tools/slack).
```

**Comment by efriis** [comment]
as a heads up, building the docs locally will yield the same broken link errors vercel does
